src/llvm/type.cr

```diff
@@ -173,7 +173,7 @@ struct LLVM::Type
           constraints.size,
           (has_side_effects ? 1 : 0),
           (is_align_stack ? 1 : 0),
-          LibLLVM::InlineAsmDialect::Intel
+          LibLLVM::InlineAsmDialect::ATT
```

Comment:
Sorry, I'm failing to see the connection to the problem at hand. Also, isn't it changing the ASM supported?

Reply:
😬 we need tests 🙈